src/config-import.js

```diff
@@ -17,24 +18,34 @@ const MAX_ICON_SIZE = 10_000_000
  *   fields:  { [id: string]: unknown }
  * }} PresetsFile
  */
+
+/** @typedef {('presets' | 'fields')} ValidDocTypes */
+/**
+ * @typedef {{
+ *  [lang: string]: unknown
+ * }} TranslationsFile
+ */
+
 /**
  * @typedef {Parameters<import('./icon-api.js').IconApi['create']>[0]} IconData
  */
 
+/** @type {Error[]} */
+let warnings = []
```

Comment:
I think I see why this got hoisted, but I don't love that this is a global. Is there a way to avoid doing something like this? Maybe by moving `translationsForLanguage` into `*translations()`?

Reply:
If two configs are imported at the same time, these warnings will get squashed because `readConfig` is asynchronous. Could we move this "back in" to `readConfig`?